R/UnConstrainedFPOPLog.R

```diff
@@ -0,0 +1,227 @@
+PeakSegFPOP <- structure(function
```

Comment:
here's the story of the Log suffix.

First I tried implemented the functional pruning using C(\mu) functions, as in the math. (where \mu is a non-negative mean parameter, for the Poisson model for non-negative count data). However I eventually realized that empirically/numerically it works better to store the functions in terms of u = \log \mu values. So I renamed everything with a Log suffix to indicate the version of the code which has that change. After testing I realized that the change was a good one, and I just deleted the old (non-Log) code.

From the perspective of the R user, the Log is just an implementation detail. So I would recommend not using that in the name of the R function.